### PR TITLE
Refactoring date picker

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**

### CODE REVIEW

File: packages/system/src/types/theme.ts [hunk 1]
```diff
@@ -101,6 +101,7 @@ export type Theme = {
       'calendar' | 'calendarCell' | 'calendarControllers',
       ComponentStyleFunction<string, string>
     >;
+    DatePicker?: Record<'actionButton', ComponentStyleFunction<string, string>>;
```
Comment (sarahgm) [inline packages/system/src/types/theme.ts:104]: why not use without parts? :)

**Comment by sarahgm** [comment]
One thing I noticed that if you try to open the Calendar in smaller view it is first thing, not responsive and the calendar is not opening

**Comment by OsamaAbdellateef** [comment]
> One thing I noticed that if you try to open the Calendar in smaller view it is first thing, not responsive and the calendar is not opening

Yeah this is true , but i tried this on the  documentation and it is working :"

**Comment by sarahgm** [comment]
> > One thing I noticed that if you try to open the Calendar in smaller view it is first thing, not responsive and the calendar is not opening
> 
> Yeah this is true , but i tried this on the documentation and it is working :"

What do you mean?

**Review by sarahgm — APPROVED** [review]

**Pull request merged**